src/app/detail/[task_id]/[user_type]/page.tsx

```diff
@@ -84,6 +84,7 @@ export default async function TaskDetailPage({
                   <Stack direction="row" alignItems="center" columnGap={3}>
                     <CustomLink
                       href={{ pathname: params.user_type === UserType.INTERNAL_USER ? `/` : `/client`, query: { token } }}
+                      revalidate={true}
                     >
```

Comment:
This is still not solving the underlying issue of caching behaviour during browser navigation. I think browser navigation part can't be ignored because users are actually navigate back the iframe through the main app. 

I think this change is not required because #280 already removes the caching by upgrading the version.